clients.yml

```diff
@@ -565,6 +565,16 @@ project/releng/generic-worker/datacenter-gecko-t-osx-1400-m2-staging:
     - queue:worker-id:mdc2/mac*
     - queue:worker-id:mdc2/t-yosemite*
     - queue:worker-id:mdc2/t-mojave*
+project/releng/generic-worker/datacenter-gecko-t-osx-1500-m4-staging:
+  description: Datacenter MacOS dedicated developer and automation worker.
+  scopes:
+    - assume:worker-type:releng-hardware/gecko-t-osx*
+    - queue:worker-id:mdc1/mac*
+    - queue:worker-id:mdc1/t-yosemite*
+    - queue:worker-id:mdc1/t-mojave*
+    - queue:worker-id:mdc2/mac*
+    - queue:worker-id:mdc2/t-yosemite*
+    - queue:worker-id:mdc2/t-mojave*
```

Comment:
Will we actually use all of these workers ids here? It looks like the M4s run Sequoia?

Reply:
I dont know the answer to the first question but yes they are running Sequoia